lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -237,13 +243,25 @@ def log.trace?
         end
 
         if @watch
+          if @metadata_source
+            self.class.send(:include, KubernetesMetadata::WatchNamespace)
```

Comment:
why are we using `self.class.send` instead of just `include KubernetesMetadata::WatchNamespace`